.editorconfig

```diff
@@ -50,9 +50,9 @@ dotnet_style_explicit_tuple_names = true:suggestion
 # CSharp code style settings:
 [*.cs]
 # Prefer "var" everywhere
-csharp_style_var_for_built_in_types = true:suggestion
+csharp_style_var_for_built_in_types = false:none
 csharp_style_var_when_type_is_apparent = true:suggestion
-csharp_style_var_elsewhere = true:suggestion
+csharp_style_var_elsewhere = false:none
```

Comment:
:memo: 'none' enforcement means that the user is free to choose to use var or not use var in the scenario. Be prepared to not comment during code reviews for the usage of var when this option is used.